docs/src/main/sphinx/client/cli.md

```diff
@@ -149,6 +149,8 @@ mode:
     processing statistics.
 * - `--disable-auto-suggestion`
   - Disables autocomplete suggestions.
+* - `--follow-redirects`
+  - If set to true, then follows client redirects. Default is false.
```

Comment:
According to the discussion in #21026, we agree to set this to true by default.

Reply:
Sure. Updating the PR to make it `true` by default.